### PR TITLE
Add a configurable assets bundle

### PR DESCRIPTION
**Pull request by VassilIordanov**
Implements #1919

### System admin changes
- it is now possible to serve files from directories configured in anet.yml

- [x] anet.yml needs change
- [x] documentation has changed

### CODE REVIEW

File: src/main/java/mil/dds/anet/AnetApplication.java [hunk 1]
```diff
@@ -160,12 +161,14 @@ public void run(AnetConfiguration configuration, Environment environment) {
     injector.getInstance(StatementLogger.class);
 
     // The Object Engine is the core place where we store all of the Dao's
-    // You can always grab the engine from anywhere with AnetObjectEngine.getInstance()
+    // You can always grab the engine from anywhere with
+    // AnetObjectEngine.getInstance()
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/AnetApplication.java]: Why is this comment wrapped?

File: src/main/java/mil/dds/anet/AnetApplication.java [hunk 2]
```diff
@@ -160,12 +161,14 @@ public void run(AnetConfiguration configuration, Environment environment) {
...
     final AnetObjectEngine engine = new AnetObjectEngine(dbUrl, this);
     environment.servlets().setSessionHandler(new SessionHandler());
 
     if (configuration.isDevelopmentMode()) {
-      // In development mode chain URL params (used during testing) and basic HTTP Authentication
+      // In development mode chain URL params (used during testing) and basic HTTP
+      // Authentication
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/AnetApplication.java]: Why is this comment wrapped?

File: src/main/java/mil/dds/anet/AnetApplication.java [hunk 3]
```diff
@@ -221,7 +224,8 @@ public void run(AnetConfiguration configuration, Environment environment) {
     scheduler.scheduleAtFixedRate(emailWorker, 5, 5, TimeUnit.MINUTES);
     scheduler.schedule(emailWorker, 5, TimeUnit.SECONDS);
 
-    // Check for any future engagements every 1 minutes in development mode or every 3 hours
+    // Check for any future engagements every 1 minutes in development mode or every
+    // 3 hours
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/AnetApplication.java]: Why is this comment wrapped?

File: src/main/java/mil/dds/anet/AnetApplication.java (hunk 1 above)
Comment (VassilIordanov) [reply src/main/java/mil/dds/anet/AnetApplication.java]: This is what `./gradlew spotlessJavaApply` does

**Pull request merged**